cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedStmt.qll

```diff
@@ -226,10 +226,11 @@ class TranslatedMicrosoftTryExceptHandler extends TranslatedElement,
 
   final override Function getFunction() { result = tryExcept.getEnclosingFunction() }
 
-  override Instruction getExceptionSuccessorInstruction(EdgeKind kind) {
+  override Instruction getExceptionSuccessorInstruction(EdgeKind kind, boolean isSEH) {
     // A throw from within a `__except` block flows to the handler for the parent of
     // the `__try`.
-    result = this.getParent().getParent().getExceptionSuccessorInstruction(kind)
+    isSEH = true and
```

Comment:
I don't think we need to restrict this to be `SEH` exceptions. I know this is totally dangerous and not supported by Microsoft, but there's nothing stopping us from reasoning about this control flow as if it was supported:
```cpp
try {
  __try {
  
  } __except(...) {
    throw MyCppException(); // this should propagate out to the enclosing try/catch
  }
catch(const MyCppException&) {
  // i.e,. to here
}
```

Reply:
Fixed